src/controllers/summarized-county.js

```diff
@@ -29,6 +29,7 @@ const modelAttributes = getModelAttributes(SummarizedCountyModel);
 const numericModelAttributes = getModelNumericAttributes(SummarizedCountyModel);
 const spotAttributes = ['state', 'county', 'year', 'spotst0'];
 const downloadFieldsToOmit = ['cleridPerDay', 'spbPerDay'];
+const downloadFieldsPrediction = ['state', 'county', 'year', 'mu', 'pi', 'expSpotsIfOutbreak', 'probSpotsGT0', 'probSpotsGT20', 'probSpotsGT50', 'probSpotsGT150', 'probSpotsGT400', 'probSpotsGT1000'];
```

Comment:
There's actually a nice trick to not repeat this across county and ranger district: if you add both county and ranger district to this array, and then move it to constants and import it, it will still work! Since you have the .includes method call in line 151, rangerDistrict will not show up in the county version, and vice versa. This is totally unnecessary when you only repeat the code once but it's fun to consider.

The version you currently have can be simplified by just putting the downloadFieldsPrediction in line 151 and taking out the modelAttributes.filter call, but simplifying that is also unnecessary.